### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by momo1986** [bbartoldson/Adversarial-Robustness-Limits#3]
Hi dear colleagues,

could you provide the pre-trained weight, or is there a script to run the python file so that I can reproduce the result on the RobustBench.

Thanks & Regards!
Momo

**Comment by bbartoldson** [comment]
Please see the following model loading and result reproduction instructions: https://github.com/RobustBench/robustbench?tab=readme-ov-file#model-zoo.

**Comment by momo1986** [comment]
OK, thanks.

If I have only one GPU in the local environment, how can I reproduce the experiment?

Thanks & Regards!
Jun

**Comment by bbartoldson** [comment]
You should be able to run the test script I linked to above with one GPU. You can also use our training code with one GPU, as far as I know (I haven't tried it), but you will likely need to add a loop to accumulate the gradient across multiple microbatches. The reason for this is you won't be able to fit the batch size we used onto one GPU -- we used DDP across hundreds of GPUs to reach our batch size without gradient accumulation steps. For the test script, this doesn't matter because you can run with any batch size without changing how the model performs.
